Sources/Private/Model/DotLottie/DotLottieAnimation.swift

```diff
@@ -20,12 +20,15 @@ struct DotLottieAnimation: Codable {
   /// 1 or -1
   var direction: Int? = 1
 
+  /// Auto plays animation
+  var autoplay: Bool? = false
```

Comment:
Do you know where I can find documentation about the json fields used in dotLottie animation manifests? That would be helpful